src/routes/usersRoute.js

```diff
@@ -1,10 +1,13 @@
 const express = require("express");
 
 const usersController = require("../controllers/usersController");
+const { checkToken } = require("../middlewares/auth")
+
 const router = express.Router();
 
 router.post("/signUp", usersController.signUp);
 router.post("/logIn", usersController.logIn);
+router.post("/:userId", checkToken, usersController.addUserAddress);
```

Comment:
userId는 endpoint로 받지 않습니다.
```suggestion
router.post("", checkToken, usersController.addUserAddress);
```